.github/dependabot.yml

```diff
@@ -5,7 +5,23 @@
 
 version: 2
 updates:
-  - package-ecosystem: "" # See documentation for possible values
+  # Maintain dependencies
+  - package-ecosystem: "npm" # See documentation for possible values
     directory: "/template/" # Location of package manifests
     schedule:
       interval: "weekly"
+    allow:
+      # Allow both direct and indirect updates for all packages
+      - dependency-type: "direct"
+    # Allow up to 10 open pull requests for pip dependencies (default: 5)
+    open-pull-requests-limit: 10
+    pull-request-branch-name:
+      # Separate sections of the branch name with a hyphen
+      # for example, `dependabot-npm_and_yarn-next_js-acorn-6.4.1`
+      separator: "-"
+    # Disable rebasing for pull requests
+    rebase-strategy: "disabled"
```

Comment:
why we not enable rebase for pull request?

Reply:
Im afraid that auto rebase will make our code lost in some case

Reply:
ok